integration/default_test.go

```diff
@@ -93,6 +93,60 @@ func testDefault(t *testing.T, context spec.G, it spec.S) {
 		})
 	})
 
+	context("when building a simple app with no dependencies but perserving files in the workspace", func() {
+		var (
+			image     occam.Image
+			container occam.Container
+
+			name   string
+			source string
+		)
+
+		it.Before(func() {
+			var err error
+			name, err = occam.RandomName()
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		it.After(func() {
+			Expect(docker.Container.Remove.Execute(container.ID)).To(Succeed())
+			Expect(docker.Volume.Remove.Execute(occam.CacheVolumeNames(name))).To(Succeed())
+			Expect(docker.Image.Remove.Execute(image.ID)).To(Succeed())
+			Expect(os.RemoveAll(source)).To(Succeed())
+		})
+
+		it("builds successfully", func() {
+			var err error
+			source, err = occam.Source(filepath.Join("testdata", "default"))
+			Expect(err).NotTo(HaveOccurred())
+
+			var logs fmt.Stringer
+			image, logs, err = pack.Build.
+				WithPullPolicy("never").
+				WithBuildpacks(
+					settings.Buildpacks.GoDist.Online,
+					settings.Buildpacks.GoBuild.Online,
+				).
+				WithEnv(map[string]string{"BP_INCLUDE_FILES": "./static-file"}).
```

Comment:
Maybe this integration test should test more than just the simplest possible case? Like using a glob and the path separator to keep 2 different paths.

Reply:
Sure thing. If that is the case then I am going to make a separate integration test for this.